<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cubes3D</class>
 <widget class="QMainWindow" name="Cubes3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cubes3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="spLeftRight">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="spTexts">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="FeEdit" name="teFeIn">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QTextEdit" name="teFeOut">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="cbAnimation"/>
        </item>
        <item>
         <widget class="QSplitter" name="spViews">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="laPixelPreview">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="View3D" name="view3d" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionclearRecent"/>
    </widget>
    <addaction name="actionnew"/>
    <addaction name="separator"/>
    <addaction name="actionopen"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>open ...</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionclearRecent">
   <property name="text">
    <string>clear</string>
   </property>
  </action>
  <action name="actionnew">
   <property name="text">
    <string>new ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View3D</class>
   <extends>QWidget</extends>
   <header>view3d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FeEdit</class>
   <extends>QTextEdit</extends>
   <header>FeEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
